boot/pod/src/boot/pod.clj

```diff
@@ -593,7 +593,8 @@
   (let [clj-dep (symbol (boot.App/config "BOOT_CLOJURE_NAME"))
         rm-clj  (if-not ignore-clj?
                   identity
-                  (partial remove #(= clj-dep (first (:dep %)))))]
+                  (partial remove #(and (= clj-dep (first (:dep %)))
+                                        (not-any? #{[:classifier "sources"]} (partition 2 (:dep %))))))]
```

Comment:
Maybe `(not= "sources" (:classifier (into {} (:dep %))))` is simpler/easier to understand?

Reply:
It looks like `into` wants pairs it can conj onto the map. We could say
```clojure
(not= "sources" (:classifier (into {} (map vec (partition 2 (:dep %))))))
```
I'd stick with the original, but up to you!